src/classlibnative/bcltype/system.cpp

```diff
@@ -73,9 +73,11 @@ FCIMPLEND;
 
 
 
-#ifndef FEATURE_CORECLR
 INT64 QCALLTYPE SystemNative::GetWorkingSet()
 {
+#if PLATFORM_UNIX
+    return 0; // TODO: Implement in PAL
```

Comment:
The interesting question is which PAL it should go to. The implementation for this exists in corefx already for Process.WorkingSet. It may be nice to reuse it. If the implementation of `Environment` class lived in corefx repo instead, what would the implementation look like - do we want to make it the same, ignoring the repo boundaries?